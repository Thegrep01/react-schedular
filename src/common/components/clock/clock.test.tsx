import React from 'react';
import Clock from './clock';
import ReactDOM from 'react-dom';


describe('Test clock component', () => {
    it('show valid time', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Clock />, div);
        ReactDOM.unmountComponentAtNode(div);
    });
});

