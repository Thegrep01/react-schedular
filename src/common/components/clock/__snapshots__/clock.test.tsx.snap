// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test clock component show valid time 1`] = `
<p
  className="MuiTypography-root MuiTypography-body1"
>
  Tue 14:01
</p>
`;
